Spring-Cloud-Gateway/Dockerfile

```diff
@@ -1,7 +1,4 @@
-FROM openjdk:8
-EXPOSE 8080
-COPY . /app
-WORKDIR /app
-ADD target/Spring-Cloud-Gateway-0.0.1-SNAPSHOT.jar spring-cloud-gateway.jar
-ENTRYPOINT ["java", "-jar", "/spring-cloud-gateway.jar"]
-CMD java -jar spring-cloud-gateway.jar
+FROM openjdk:16
```

Comment:
are you sure upgrading to java 16 will not cause any issues? check with the devs on this one.

Reply:
Their project requires Java 16 to compile. I've tested the docker images in both docker-compose and kubernetes and it seems to be working fine. 

If anything, I would be worried about a java 8 image being able to run the jar files. I wasn't able to get their docker images working when I first started.

Reply:
i guess the dockerfiles were just out of date with how they were doing the rest.